prometheus/promhttp/http.go

```diff
@@ -11,21 +11,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Copyright (c) 2013, The Prometheus Authors
-// All rights reserved.
-//
-// Use of this source code is governed by a BSD-style license that can be found
-// in the LICENSE file.
-
-// Package promhttp contains functions to create http.Handler instances to
-// expose Prometheus metrics via HTTP. In later versions of this package, it
-// will also contain tooling to instrument instances of http.Handler and
-// http.RoundTripper.
-//
+// Package promhttp provides new functions for instrumenting go code.
 // promhttp.Handler acts on the prometheus.DefaultGatherer. With HandlerFor,
 // you can create a handler for a custom registry or anything that implements
 // the Gatherer interface. It also allows to create handlers that act
 // differently on errors or allow to log errors.
+//
+// The package also contains functions to create http.Handler instances to
+// expose Prometheus metrics via HTTP. It contains tooling to instrument
```

Comment:
Scratch "functions to create http.Handler instances to expose Prometheus metrics via HTTP. It contains", i.e. just write: "The package also contains tooling to instrument..."